composer.json

```diff
@@ -14,7 +14,8 @@
         }
     ],
     "require": {
-        "mockery/mockery": "~0.8"
+        "mockery/mockery": "~0.8",
+        "codeception/codeception": "^2.2"
```

Comment:
I think we can set this to "^2.0". The module should be compatible even with Codeception <2 but there is no need to support that now.